packages/polaris-viz/src/components/Grid/Grid.tsx

```diff
@@ -323,13 +342,12 @@ export function Grid(props: GridProps) {
           setXAxisHeight={setXAxisHeight}
         />
       </svg>
-      {tooltipInfo && (
-        <Tooltip
-          x={tooltipInfo.x}
-          y={tooltipInfo.y}
-          group={groupSelected || hoveredGroup}
-        />
-      )}
+      <Tooltip
+        ref={tooltipRef}
+        x={tooltipInfo?.x ?? 0}
```

Comment:
I think we can remove the ?? and bring back the `tooltipInfo &&` I think we only removed it when trying to calculate the height